pandas/tseries/tests/test_tslib.py

```diff
@@ -1282,6 +1282,46 @@ def test_resolution(self):
                 self.assertEqual(result, expected)
 
 
+class TestTimedelta(tm.TestCase):
+    def test_min(self):
```

Comment:
these are completely in the wrong place. they should go in ``tseries/tests/test_timedeltas.py``